UNRELEASED.md

```diff
@@ -8,6 +8,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Enhancements
 
+- Allow type override for components, ex: StackItem...
```

Comment:
Do we want to publicly call this out? Should it be an internal feature?

Reply:
Good call. I will skip the changelog for this PR.